test/node/bigint.test.ts

```diff
@@ -1,89 +1,124 @@
 import { BSON } from '../register-bson';
 import { bufferFromHexArray } from './tools/utils';
 import { BSON_DATA_LONG } from '../../src/constants';
+import { BSONDataView } from '../../src/utils/byte_utils'
 
-describe('BSON BigInt serialization Support', function () {
-  // Index for the data type byte of a BSON document with a single element
-  const DATA_TYPE_OFFSET = 4;
+describe('BSON BigInt serialization Support', function() {
+  // Index for the data type byte of a BSON document with a 
+  // NOTE: These offsets only apply for documents with the shape {a : <n>}
+  // where n is a BigInt
+  class SerializedDocParts {
```

Comment:
```suggestion
  type SerializedDocParts = {
```

Since we're not actually using the class we can make this a TS only description of the object shape